admin/data/migrations/migrations/20171012121301-add-new-attendance-codes.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const config = require('../config')
+const AttendanceCode = require('../../../models/attendance-code')
+
+module.exports = {
+  up (db, next) {
+    mongoose.connect(config.mongodb.url, async error => {
+      if (error) {
+        next(new Error('Could not connect to mongodb: ' + error.message))
+      }
+
+      try {
+        db.collection('attendancecodes').drop()
+        await createAttendanceCode('Incorrect registration', 1, 0)
```

Comment:
To be absolutely on it, the drop() almost certainly takes a callback, so the awaits should sit inside the callback.  Otherwise fine :)